website/client/src/components/shops/quests/questRewards.vue

```diff
@@ -0,0 +1,213 @@
+<template>
+  <div class="quest-rewards">
+    <div
+      class="header d-flex align-items-center"
+      @click="toggle"
+    >
+      <span class="d-flex justify-content-center">
+        <div
+          v-once
+          class="your-rewards d-flex align-items-center"
+        >
+          <span
+            class="sparkles"
+            v-html="icons.sparkles"
```

Comment:
`v-once`, as usual

Reply:
some lines higher the element is already [`v-once`](https://github.com/HabitRPG/habitica/pull/13112/files/f028bcbc4b674cfabd3f83f9863766fe5b27e93d#diff-d41252d940211986ef3af224b38110c17263dfe343458c9edb41a48edf6fd2a0R9)